distributed/asyncio.py

```diff
@@ -117,17 +108,15 @@ class AioClient(Client):
     """
     _Future = AioFuture
 
-    def __init__(self, *args, loop=None, start=True, set_as_default=False,
-                 **kwargs):
```

Comment:
If you're using this from an asyncio application, it's your job to start the loop, not AioClient's. Furthermore, both asyncio and Tornado are rapidly moving away from this idiom where you pass a "loop" argument. Tornado just deleted all "ioloop" arguments on master, and Yury said yesterday that asyncio is deleting most "loop" arguments circa Python 3.7.